libsolidity/parsing/Parser.cpp

```diff
@@ -679,6 +689,7 @@ ASTPointer<VariableDeclaration> Parser::parseVariableDeclaration(
 
 	bool isIndexed = false;
 	bool isDeclaredConst = false;
+	bool isVirtual = false;
```

Comment:
You could still consider to just remove these changes from this PR instead of adding this here only to remove it in a follow-up PR... but as you like!